src/context/PlannerContext.jsx

```diff
@@ -1,21 +1,32 @@
-import { createContext, useContext, useEffect, useReducer } from 'react';
+import {
+  createContext,
+  useContext,
+  useEffect,
+  useReducer,
+  useState,
+} from 'react';
 import { parseDate } from '../utils/parseDate';
 
 // payload is an entry object:
 // { title: String, content: String, date: Date }
 function entriesReducer(entries, { type, payload }) {
+  console.log('payload', payload);
   switch (type) {
     case 'create':
       const entry = { ...payload, id: entries.length };
-      return [entry, ...entries];
+      const updatedEntries = [entry, ...entries];
+      localStorage.setItem('ENTRIES', JSON.stringify(updatedEntries));
```

Comment:
We should keep our reducers pure and only concerned with updating state. I would recommend moving these local storage functions outside of the reducer

Reply:
See the "Writing Reducers Well" portion of the docs:
https://beta.reactjs.org/learn/extracting-state-logic-into-a-reducer